.github/ISSUE_TEMPLATE.md

```diff
@@ -1,36 +1,64 @@
-<!--
-Use Markdown language for styling.
-Github's Markdown Guide https://guides.github.com/features/mastering-markdown/
-Please use the SEARCH function before opening a new issue!
+<!-- 
+Make sure that the issue you're about to make does not exist already, use the search function to find possible duplicates.
+
+*OBS: Reports that does not follow the template below, will be considered invalid and closed. :OBS*
```

Comment:
What is OBS supposed to be? It is not clear.

Reply:
I was referring to Observe but I think I'll just remove it.